en/flight_modes/return.md

```diff
@@ -117,29 +130,28 @@ If no mission is defined PX4 will fly directly to home location and land (rally
 
 If the mission changes during return mode, then the behaviour is re-evaluated based on the new mission following the same rules as above (e.g. if the new mission has no landing sequence and you're in a mission, the mission is reversed).
 
-<a id="safety_point_return"></a>
-
 ### Closest Safe Destination Return Type (RTL_TYPE=3)
 
 In this return type the vehicle:
 
-- Ascends to a safe [return altitude](#return_altitude) (above any expected obstacles).
+- Ascends to a safe [minimum return altitude](#return_altitude) (above any expected obstacles).
 - Flies a direct path to the closest destination of: home location, mission landing pattern or rally point.
-- If the destination is a mission landing pattern the vehicle will follow the pattern to land
-- If the destination is a home location or rally point, the vehicle will descend to the descent altitude ([RTL_DESCEND_ALT](#RTL_DESCEND_ALT)) and then [Land or waits](#arrival).
+- If the destination is a [mission landing pattern](#mission-landing-pattern) the vehicle will follow the pattern to land.
+- If the destination is a home location or rally point, the vehicle will descend to the descent altitude ([RTL_DESCEND_ALT](#RTL_DESCEND_ALT)) and then [Land or waits](#hover-landing-at-destination).
 
 <a id="return_altitude"></a>
 
-## Return Altitude
+## Minimum Return Altitude
 
-A vehicle will usually first ascend to a safe altitude before returning, in order to avoid any obstacles between it and the destination.
+For most [return types](#return_types) a vehicle will ascend to a _minimum safe altitude_ before returning (unless already above that altitude), in order to avoid any obstacles between it and the destination.
 
 :::note
-This is true for most [return types](#return_types).
-The exception is when executing a [mission path return](#mission_path_return) from within a mission, where the vehicle follows mission waypoints (we can assume these avoid any obstacles).
+The exception is when executing a [mission path return](#mission-path-return-type-rtl-type-2) from _within a mission_.
+In this case the vehicle follows mission waypoints, which we assume are planned to avoid any obstacles.
 :::
 
 The return altitude for a fixed-wing vehicle is configured using the parameter [RTL_RETURN_ALT](#RTL_RETURN_ALT).
+
 The return altitude for multicopter and VTOL vehicles is configured using the parameters [RTL_RETURN_ALT](#RTL_RETURN_ALT) and [RTL_CONE_ANG](#RTL_CONE_ANG), which define a half cone centered around the destination (home location or safety point).
```

Comment:
@sfuhrer Here we imply that the fixed wing return altitude does not use the code. Is it that it "cannot" or that "it does not by default". If it is the later, is it that it "should not"?

Reply:
It cannot use it.